pkg/build/node/const.go

```diff
@@ -24,225 +24,202 @@ const (
+metadata:
+  name: kindnet
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kindnet
+subjects:
+- kind: ServiceAccount
+  name: kindnet
+  namespace: kube-system
+---
 apiVersion: v1
-kind: List
-items:
-  - apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: weave-net
-      labels:
-        name: weave-net
-      namespace: kube-system
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    kind: ClusterRole
-    metadata:
-      name: weave-net
-      labels:
-        name: weave-net
-    rules:
-      - apiGroups:
-          - ''
-        resources:
-          - pods
-          - namespaces
-          - nodes
-        verbs:
-          - get
-          - list
-          - watch
-      - apiGroups:
-          - extensions
-        resources:
-          - networkpolicies
-        verbs:
-          - get
-          - list
-          - watch
-      - apiGroups:
-          - 'networking.k8s.io'
-        resources:
-          - networkpolicies
-        verbs:
-          - get
-          - list
-          - watch
-      - apiGroups:
-        - ''
-        resources:
-        - nodes/status
-        verbs:
-        - patch
-        - update
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    kind: ClusterRoleBinding
-    metadata:
-      name: weave-net
-      labels:
-        name: weave-net
-    roleRef:
-      kind: ClusterRole
-      name: weave-net
-      apiGroup: rbac.authorization.k8s.io
-    subjects:
-      - kind: ServiceAccount
-        name: weave-net
-        namespace: kube-system
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    kind: Role
-    metadata:
-      name: weave-net
-      namespace: kube-system
-      labels:
-        name: weave-net
-    rules:
-      - apiGroups:
-          - ''
-        resources:
-          - configmaps
-        resourceNames:
-          - weave-net
-        verbs:
-          - get
-          - update
-      - apiGroups:
-          - ''
-        resources:
-          - configmaps
-        verbs:
-          - create
-  - apiVersion: rbac.authorization.k8s.io/v1beta1
-    kind: RoleBinding
-    metadata:
-      name: weave-net
-      namespace: kube-system
-      labels:
-        name: weave-net
-    roleRef:
-      kind: Role
-      name: weave-net
-      apiGroup: rbac.authorization.k8s.io
-    subjects:
-      - kind: ServiceAccount
-        name: weave-net
-        namespace: kube-system
-  - apiVersion: extensions/v1beta1
-    kind: DaemonSet
+kind: ServiceAccount
+metadata:
+  name: kindnet
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kindnet-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: kindnet
+data:
+  cni-conf-template.json: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "kindOfBridge",
+      "plugins": [
+        {
+          "type": "bridge",
+          "ipMasq": true,
+          "isDefaultGateway": true,
+          "hairpinMode": true,
+          "ipam": {
```

Comment:
I guess we should add the dataDir option 

> Allocated IP addresses are stored as files in /var/lib/cni/networks/$NETWORK_NAME. The path can be customized with the dataDir option listed above. Environments where IPs are released automatically on reboot (e.g. running containers are not restored) may wish to specify /var/run/cni or another tmpfs mounted directory instead.
https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md

Reply:
done